backend/Dockerfile

```
@@ -2,10 +2,16 @@ FROM namely/protoc-all as generator
 WORKDIR /
 COPY protos protos
 COPY backend backend
-RUN protoc --proto_path=protos protos/* --python_out=backend/protos 
+RUN ln $(which grpc_python_plugin) /usr/bin/protoc-gen-grpc_python
+RUN protoc --proto_path=protos protos/* --python_out=backend/protos --grpc_python_out=backend/protos
```

```suggestion
RUN protoc --proto_path=. protos/* --python_out=backend --grpc_python_out=backend
```

Voir https://github.com/PolyCortex/polydodo/pull/24#discussion_r496328753